lib/pages/share/share_view.dart

```diff
@@ -0,0 +1,84 @@
+import 'package:fluffychat/pages/forward/recent_chat_list.dart';
+import 'package:fluffychat/pages/forward/recent_chat_title.dart';
+import 'package:fluffychat/pages/share/share.dart';
+import 'package:fluffychat/widgets/matrix.dart';
+import 'package:fluffychat/widgets/twake_components/twake_fab.dart';
+import 'package:fluffychat/widgets/twake_components/twake_icon_button.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/l10n.dart';
+import 'package:go_router/go_router.dart';
+import 'package:matrix/matrix.dart';
+
+class ShareView extends StatelessWidget {
+  const ShareView(this.controller, {super.key});
+
+  final ShareController controller;
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        centerTitle: true,
+        title: Text(
+          L10n.of(context)!.share,
+          style: Theme.of(context).textTheme.headlineLarge,
+        ),
+        leading: TwakeIconButton(
+          tooltip: L10n.of(context)!.cancel,
+          icon: Icons.close,
+          onPressed: () => context.go('/rooms'),
```

Comment:
Why do you use `context.go`?
Why don't you use `context.pop`?

Reply:
because in go_router, it not a child of `rooms`